build/run-code-lint.sh

```diff
@@ -22,7 +37,7 @@ gem install mdl
 gem install awesome_bot
 
 # Install golangci-lint
-curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)"/bin v1.20.0
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b "$(go env GOPATH)"/bin v1.28.3
```

Comment:
Using the newer golangci could cause more lint errors in the code repo. That could spent more time to clean them up. I would suggest to stick to 1.20 to avoid lint check failure

Reply:
I added a new lint script that is only use for prow so we can still have our travis build for now while we resolve our linting issue.